client/src/components/ElementEditor/Header.js

```diff
@@ -1,14 +1,55 @@
 import React, { Component, PropTypes } from 'react';
+import { Tooltip } from 'reactstrap';
 
 class Header extends Component {
+  constructor(props) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      tooltipOpen: false
+    };
+  }
+
+  toggle() {
+    this.setState({
+      tooltipOpen: !this.state.tooltipOpen
+    });
+  }
+
   render() {
+    const { id, title, elementType, fontIcon } = this.props;
+
     return (
-      <div />
+      <div className="element-editor-header">
+        <div className="element-editor-header-info">
```

Comment:
I think `element-editor-header__info` would be more BEM

Reply:
Yeah, I have seen it written like that a few times before, indeed. It's confusing, as http://getbem.com/introduction/ quotes that particular style:
![image](https://user-images.githubusercontent.com/14869519/43436881-71109e40-94da-11e8-9e19-8aacd08e7aa3.png) (Which I am aware I haven't been following strictly either). Your call, shall I follow BEM BEM or SilverStripe BEM?!

Reply:
SilverStripe BEM === BEM 😄 (or should be at least)

In this case `element-editor-header` is the block, and `info`, `title`, `icon` etc is the element. See http://getbem.com/naming/